storage/Utils/LockImpl.cc

```diff
@@ -66,8 +66,10 @@ namespace storage
 		y2err("creating directory for lock-file failed: " << strerror(errno));
 	    }
 
-	    fd = open(LOCK_DIR "/lock", (read_only ? O_RDONLY : O_WRONLY) | O_CREAT | O_CLOEXEC,
-		      S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH);
+	    const int permissions = S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH; // 0644
+
+	    fd = open(LOCK_DIR "/lock", (read_only ? O_RDONLY : O_WRONLY) | O_CREAT | O_CLOEXEC, permissions);
```

Comment:
Since you already have a variable for those permissions, why change them here again?
Why not work on that _permissions_ variable before this line and then just pass _permissions_ to this _open()_ call? This would be much better to read and much better maintainable.

Reply:
`open()` is called with three parameters, and permissions is the last one. The second parameter is the `oflags`, and I have left it untouched.

Reply:
OK. Silly me.